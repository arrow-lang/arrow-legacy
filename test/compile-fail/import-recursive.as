import recursiveA from "./fixtures/recursive-a";
import recursiveB from "./fixtures/recursive-b";
