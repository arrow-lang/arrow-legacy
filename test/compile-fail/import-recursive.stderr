[0;37mtest/compile-fail/fixtures/recursive-b.as:1.1-31: [0;31merror: [1;37munresolvable circular dependency with module: 'recursive-a'[0m
[0;37mtest/compile-fail/fixtures/recursive-a.as:1.1-31: [0;31merror: [1;37munresolvable circular dependency with module: 'recursive-b'[0m
